### PR TITLE
update "API" class instantiation calls in example scripts

### PR DESCRIPTION
**Pull request by agalileev**
Hi. It looks like my very first commit belongs to you. :-)
I started my research on wiki-scripts and found out, that "diff-revisions.py" example is not working because of API class constructor interface was changed.
More commits are coming. (:

### CODE REVIEW

File: examples/diff-revisions.py [hunk 1]
```diff
@@ -3,17 +3,21 @@
 import os.path
 
 from ws.client import API
+from ws.client.connection import Connection
 from ws.diff import RevisionDiffer
 
 api_url = "https://wiki.archlinux.org/api.php"
+index_url = "https://wiki.arclinux.org/index.php"
 cookie_path = os.path.expanduser("~/.cache/ArchWiki.cookie")
+session = Connection.make_session(ssl_verify=True,
```
Comment (lahwaacz) [inline examples/diff-revisions.py]: You can use `API.make_session` because `API` is a subclass of `Connection`.

**Comment by lahwaacz** [comment]
Thanks, the examples are indeed outdated.

File: examples/diff-revisions.py (hunk 1 above)
Comment (agalileev) [reply examples/diff-revisions.py]: Done.

**Comment by agalileev** [comment]
Actually i want to start with fully updating examples to better understand how the wiki-scripts work.
I should warn that I'm a totally newbie, so any comments on my codestyle/code quality/git and github using are appreciated.

**Comment by lahwaacz** [comment]
No problem, looking forward to your updates :+1:

**Comment by agalileev** [comment]
I fixed the API() calls in the example scripts where it was needed.
The main thing I understood - it was a bad idea to make the pull request from master to master. :(
Some examples scripts are not fixed yet (there are an import error), but I need to read some fine manual about Git. All API() calls are fixed, so if it possible I suggest doing the merge. I want to make further fixes in separate branch on my local repository...

**Comment by lahwaacz** [comment]
OK, thanks. If you find something broken and incomprehensible, feel free to ask about it any time - better than losing time trying to understand it...

**Pull request merged**